scripts/ci/gpr-delete.mjs

```
@@ -95,21 +101,70 @@ async function findPackageApiName(owner, unscopedName, token) {
   return null;
 }
 
+async function deleteVersionById(root, versionId, scopedName, versionName, token) {
+  const response = await githubApi(`${root}/versions/${versionId}`, { method: "DELETE", token });
+  if (response.status === 204) {
+    console.error(`Deleted ${scopedName}@${versionName}`);
```

ESLint

[error] Unexpected console statement.

(no-console)